modules/installation-vsphere-regions-zones-host-groups.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+//* installing/installing-vsphere-installer-provisioned-customizations.adoc [IPI]
+//* installing/installing-vsphere-installer-provisioned-network-customizations.adoc [IPI]
+//* installing/installing-restricted-networks-installer-provisioned-vsphere.adoc [IPI]
+
+:_mod-docs-content-type: CONCEPT
+[id="installation-vsphere-regions-zones-host-groups_{context}"]
+= VMware vSphere host group enablement
+
+:FeatureName: Openshift zones support for vSphere host groups
+include::snippets/technology-preview.adoc[]
+
+When deploying an {product-title} cluster to vSphere, you can map your vSphere host groups onto {product-title} failure domains. This is useful if you are using a stretched cluster configuration, where ESXi hosts are grouped into host groups by physical location.
+
+To enable this feature:
+
+* You must arrange your ESXi hosts into host groups.
+* You must create a vCenter tag in the `openshift-region` tag category for your cluster. After you create the tag, you must attach the tag to the cluster.
+* You must create a vCenter tag in the `openshift-zone` tag category for each host group and then attach the correct tag to each ESXi host.
+* You must define multiple failure domains for your {product-title} cluster in the `install-config.yaml` file.
+* You must grant the `Host.Inventory.EditCluster` privilege on the vSphere vCenter Cluster object.
```

Comment:
Should Cluster be capitalized here? I see a few instances of vSphere vCenter cluster in our docs. And no references to vSphere vCenter Cluster object.